.github/workflows/main.yml

```diff
@@ -74,11 +74,11 @@ jobs:
           - protocol: Nairobi
             testnet: nairobinet
             testnet_uppercase: NAIROBINET
-            flextesa_docker_image: oxheadalpha/flextesa:20230607
+            flextesa_docker_image: oxheadalpha/flextesa:latest
           - protocol: Oxford
             testnet: oxfordnet
             testnet_uppercase: OXFORDNET
-            flextesa_docker_image: oxheadalpha/flextesa:20230915
+            flextesa_docker_image: oxheadalpha/flextesa:latest
```

Comment:
I'm still on the fence about changing these to point to `latest`

Reply:
and also, nairobinet and oxfordnet cannot both point to `latest`. The Nairobinet docker image is not going to use a nairobibox if we point to latest.

Reply:
I think even the latest, when called with `NairobiBox` param will run NairobiNet